server/__init__.py

```diff
@@ -226,6 +225,19 @@ def register_routes_common(app):
 def create_app(nl_root=DEFAULT_NL_ROOT):
   app = Flask(__name__, static_folder='dist', static_url_path='')
 
+  cfg = lib_config.get_config()
+
+  if not cfg.LOCAL:
+    client = google.cloud.logging.Client()
+    client.setup_logging()
+  logging.basicConfig(
```

Comment:
future PR is fine, should we do this logging config in a util and reuse from everywhere

Reply:
The cfg is for the web server only. Now nl does not have it. Should look into a general way to check if running in a GCP environment..